app/views/reviews/new.html.erb

```diff
@@ -0,0 +1,16 @@
+<h2>Add New Review</h2>
+<section>
+  <%= form_tag("/shelters/#{@shelter.id}/reviews", method: :post)  do %>
+
+    <%= label_tag "Title" %>
+    <%= text_field_tag :title %><br/>
+    <%= label_tag "Rating" %>
+    <%= text_field_tag :rating %><br/>
+    <%= label_tag "Content" %>
+    <%= text_field_tag :content %><br/>
+    <%= label_tag "Image (optional)" %>
```

Comment:
I am curious what the calculus is to handle the images as the last step of the form... A possible consideration is that if later down the road we are going to deal with file transmission that we may want to start that process immediately...